### PR TITLE
Extra templating for the index.html

### PR DESCRIPTION
**Pull request by mkeeler**

### CODE REVIEW

File: GNUmakefile [hunk 1]
```diff
@@ -1,8 +1,8 @@
 SHELL = bash
 GOGOVERSION?=$(shell grep github.com/gogo/protobuf go.mod | awk '{print $$2}')
 GOTOOLS = \
-	github.com/elazarl/go-bindata-assetfs/go-bindata-assetfs \
-	github.com/hashicorp/go-bindata/go-bindata \
+	github.com/elazarl/go-bindata-assetfs/go-bindata-assetfs@master \
+	github.com/hashicorp/go-bindata/go-bindata@master \
```
Comment (johncowen) [inline GNUmakefile]: Didn't these get pinned to a git commit (so not master) a while ago? I'm sure I've seen these with a specific commit on the end?

Comment (mikemorris) [reply GNUmakefile]: Guessing you're thinking of https://github.com/hashicorp/consul/pull/6073, thoughts here @alvin-huang?

Comment (alvin-huang) [reply GNUmakefile]: Yeah, `@master` is fine though, it was missing commits past the latest tag that we needed for the build process. This PR can go in as is and I will rebase #6073 to pin the commits.

**Comment by mikemorris** [comment]
Tested this locally, looks good to me 👍

**Pull request merged**